manifests/libreoffice/libreoffice.yaml

```diff
@@ -2,10 +2,14 @@ id: libreoffice
 name: LibreOffice
 version: 6.2.0.3
 home: https://www.libreoffice.org/
+repo: https://cgit.freedesktop.org/libreoffice
 installMethod: MSI
 installers:
+args:
```

Comment:
your yaml format is a bit off, `args` needs to be above `installers`, the way you have it right now all installers are children of args rather than `installers`

Reply:
Oops. Easy fix.